workflow/controller/operator.go

```diff
@@ -262,6 +262,33 @@ func (woc *wfOperationCtx) operate() {
 		} else {
 			woc.globalParams[common.GlobalVarWorkflowStatus] = string(workflowStatus)
 		}
+
+		var failures []interface{}
+		for _, node := range woc.wf.Status.Nodes {
+			if node.Phase == wfv1.NodeFailed || node.Phase == wfv1.NodeError {
+				failures = append(failures, struct {
+					Name         string `json:"name"`
+					Message      string `json:"message"`
+					TemplateName string `json:"templateName"`
+					FinishedAt   string `json:"finishedAt"`
+					Phase        string `json:"phase"`
+				}{
+					Name:         node.DisplayName,
+					Message:      node.Message,
+					TemplateName: node.TemplateName,
+					FinishedAt:   node.FinishedAt.String(),
```

Comment:
what timezone is this in?

Reply:
It is the same information from `NodeStatus`, so whichever timezone that is in